Source/CarthageKit/Project.swift

```diff
@@ -631,6 +648,17 @@ private func filesInDirectory(directoryURL: NSURL, typeIdentifier: String) -> Si
 		}
 }
 
+/// Sends the URL to each file found in the given directory that has the
+/// given file extension.
+///
+/// Note: If a proper UTI is defined for the file extension,
+///	      `filesInDirectory(_:typeIdentifier:)` is preferred.
+private func filesInDirectory(directoryURL: NSURL, #fileExtension: String) -> SignalProducer<NSURL, CarthageError> {
+	return NSFileManager.defaultManager().carthage_enumeratorAtURL(directoryURL, includingPropertiesForKeys: [], options: .SkipsHiddenFiles | .SkipsPackageDescendants, catchErrors: true)
```

Comment:
"bcsymbolmap" files have no defined UTI, and it turns out that if I generate a UTI from the file extension "bcsymbolmap" it doesn't match the UTI that Spotlight actually comes up with. If I generate a UTI that conforms to "public.data" then it matches, but I don't want to rely on that. Therefore, we need to test file extension directly instead.